pkg/controller/plan/validation.go

```diff
@@ -792,14 +814,44 @@ func (r *Reconciler) validateVddkImage(plan *api.Plan) (err error) {
 			r.Log.Info("validation of VDDK job is in progress", "image", image)
 			plan.Status.SetCondition(vddkValidationInProgress)
 		}
+		var ctx *plancontext.Context
+		ctx, err = plancontext.New(r, plan, r.Log)
+		if err != nil {
+			break
+		}
+		pods := &core.PodList{}
+		if err = ctx.Destination.Client.List(context.TODO(), pods, &client.ListOptions{
+			Namespace:     plan.Spec.TargetNamespace,
+			LabelSelector: k8slabels.SelectorFromSet(map[string]string{"job-name": job.Name}),
```

Comment:
cool, now this call would work in all scenarios but I think it should be placed elsewhere since in many reconcile-cycles we'll find the job in `JobComplete` and won't need the information from the pod